### PR TITLE
Editorial: Add Changelog for 1.3

### PR DESCRIPTION
**Pull request by jnurthen**
Adds Changelog to 1.3

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/2096.html" title="Last updated on Jan 3, 2024, 11:35 PM UTC (f393ca2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/2096/a0f796f...f393ca2.html" title="Last updated on Jan 3, 2024, 11:35 PM UTC (f393ca2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14088,75 +14097,236 @@ <h2>Substantive changes targeted for the 1.3 release</h2>
...
-			<li>11-Oct-2019: Deprecate <rref>directory</rref> role</li>
-			<li>11-Oct-2019: Make <rref>form</rref> role accessible name required true</li>
-			<li>11-Oct-2019: Remove allowance of <rref>group</rref> in <rref>list</rref></li>
-            <li>09-Oct-2019: Add missing implicit value for <rref>progressbar</rref></li>
-			<li>09-Oct-2019: Remove accessible name required from <rref>log</rref> and <rref>timer</rref></li>
-			<li>22-Aug-2019: Remove <pref>aria-level</pref> from <rref>grid</rref></li>
-			<li>23-Jul-2019: Add <rref>generic</rref> role</li>
-			<li>11-Jul-2019: Remove advice against changing roles</li>
-			<li>11-Jul-2019: Set Accessible Name Required to false on <rref>gridcell</rref></li>
-			<li>11-Jul-2019: Add <rref>associationlist</rref>, <rref>associationlistitemkey</rref>, and <rref>associationlistitemvalue</rref> roles</li>
```
Comment (cookiecrook) [inline index.html:14123]: will need to remove this with #2095

**Review by cookiecrook — COMMENTED** [review]
@jnurthen I'm mildly concerned (I'll get over it) that we're taking a reasonably readable changelog, and replacing it with a list of often-vague and difficult-to-change-later commit messages that contributors may not expect to be tracked as the official changelog...

For example, several below are vague:
- "brailleroledescription update" (what was updated?)
- "aria-atomic has no default" (was the default added or removed?)

That said, I can see it's desirable to have the auto-generated list from ReSpec... 

Perhaps @spectranaut has some ideas about how to convey this new expectation about commit messages to contributors.

**Comment by cookiecrook** [comment]
No objection to merging as-is... Just postulating about how we can change our practices to keep improve the 1.4 and later changelogs... Letting all editors and regular contributors know about this expectation would be a good first step.

**Comment by jnurthen** [comment]
@cookiecrook It is up to the editors to ensure the PRs have a good title before we merge them (we use the PR title for the merge commit). Many of these pre-date us thinking this was a possibility. Hopefully the next time will be less messy than this.

**Pull request merged**